tests-stress/locust/README.md

```diff
@@ -0,0 +1,55 @@
+# Stress tests manual
+
+This manual is meant for the Locust stress tests located in the same folder than this README.
+
+## Goals
+
+The following goals are meant with the stress-tests:
+
+* Monitor behavior of a production-like environment under similar stress.
+* Preemptively discover technical issues by overloading our staging environment.
+* Fix discovered issues in the production-like environment and propagate to production.
+
+Our stress-tests can also act as load-tests that are ran against our build pipeline in a daily manner at minimum:
+
+* Ensure our system can take the expected daily traffic of notifications that we receive.
+* Align and certify our SLA/SLO/SLI agreements negotiated with our clients.
+* Discover regressions related to performance that new changes can affect on our code base and infrastructure.
+
+## How to configure the stress tests
+
+There is an override system that [Locust implements with configuration parameters](https://docs.locust.io/en/stable/configuration.html). It can read values from the command-line, environment variables or custom configuration file. The order is, as defined by its own documnentation:
+
+```doc
+~/locust.conf -> ./locust.conf -> (file specified using --conf) -> env vars -> cmd args
```

Comment:
What is taking precedence here? Is `->` better understood than `>`?

Reply:
I copied/pasted from the official documentation here which states in which order the values will be read, hence latest will override precedent category.